src/python/WMCore/ReqMgr/Service/Request.py

```diff
@@ -36,6 +40,14 @@ def __init__(self, app, api, config, mount):
         # this need for the post validtiaon
         self.gq_service = WorkQueue(config.couch_host, config.couch_workqueue_db)
 
+        # initialize NATS if requested
+        self.nats = None
+        if hasattr(config, 'use_nats') and config.use_nats and hasattr(config, 'nats_server'):
+            topic = 'cms.reqmgr2'
```

Comment:
Should we make it configurable? Or it's very likely that this is not going to change in the next few years?

Reply:
This is the *main* default topic. I'd like to keep them fixed, `cms.reqmgr2` for ReqMgr2, `cms.dbs` for DBS, `cms.wmarchive` for WMArchive. Therefore I don't see needs to configure it here.

Reply:
within the main topic the messages may be divided into sub-topics, e.g. I already have `cms.wmarchive.site`, `cms.wmarchive.exitCodes`, etc. Therefore I would expect that in ReqMgr2 we'll define sub-topics too and those may be configurable.